exec_test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+#Make it executable by default (might not be needed)
+chmod +x exec_test.sh
+
+function argsHelp()
+{
+    echo -e "\n *** Usage help ***"
+    echo -e "\n $0 [-t path/to/testcase.py] \n"
+    exit 1
+}
+while getopts ":t:" flag
+do
+    case "$flag" in 
+        t) testcase=${OPTARG} ;;
+        :) argsHelp ;;
+        \?) argsHelp ;;
+    esac
+done
+
+#Check if t param is missing
+if [ -z "$testcase" ]
+then
+    echo -e "\n ### Command line param missing ! ###"
+    argsHelp
+fi
+
+#Build the mta docker image from dockerfile
+docker build dockerfiles/docker_fedora31/
+
+#Run the mta container on port 8080 in detached mode
+if [[ $(docker ps | grep windup_mta) = *windup_mta* ]]
+then
+    echo -e "\n MTA web console container is already running !"
+else
+    if [[ $(docker ps -a | grep windup_mta) = *windup_mta* ]]
+    then
+        echo -e "\n Starting the existing MTA web console docker container !"
+        docker start windup_mta
+    else
+        echo -e "\n Starting new MTA web console container !"
+        docker run -d -p 8080:8080 --name windup_mta -it mta
+    fi
+fi
+
+#Wait for mta web console to be fully up
+echo -e "\n Waiting for mta web console to be available ..."
+count=0
+threshold=3
+until $(curl --output /dev/null --silent --head --fail http://localhost:8080/mta-web); do
+    if [ ${count} == ${threshold} ]
+    then
+      echo -e "\n Could not reach web console after all tries, exiting ..."
+      exit 1
+    fi
+    echo ' \....\ '
+    count=$(($count+1))
+    sleep 10
+done
+
+echo -e "\n MTA web console is available now ! \n"
+
+#Setup python3 virtual environment (version issue needs to be sorted)
+python3.7 -m venv .mta_venv
+source ./.mta_venv/bin/activate
+pip install -e .
+
+#Start selenium container
+if [[ $(docker ps | grep mta_selenium) = *mta_selenium* ]]
```

Comment:
I don't think we need if else here.
If we do "mta selenium start" and the server is already running it just gives message "server already running "